python/shark_turbine/dynamo/importer.py

```diff
@@ -94,6 +100,24 @@
     torch.complex128: "complex<f64>",
 }
 
+TORCH_DTYPE_TO_IREE_TYPE: Dict[torch.dtype, Callable[[], MlirType]] = {
```

Comment:
We're going to be upstreaming the importer, so let's not refer to IREE. Change this to TORCH_DTYPE_TO_MLIR_TYPE